### PR TITLE
TLT-4461 Template support and Client Services feedback

### PR DESCRIPTION
**Pull request by Chris-Thornton-Harvard**
Takes care of some of the items listed in the testing session doc https://docs.google.com/document/d/1a1uS0bi52KZazKfutdyw_ez7hCrKUGorIn-6qr8nuzw/edit and adds the template name to the job detail page

### CODE REVIEW

File: bulk_site_creator/views.py [hunk 1]
```diff
@@ -178,6 +181,7 @@ def create_bulk_job(request: HttpRequest) -> Optional[JsonResponse]:
     school_id = sis_account_id.split(":")[1]
 
     table_data = json.loads(request.POST['data'])
+    print(f'Table Data: {table_data}')
```
Comment (PatrickMTonne) [inline bulk_site_creator/views.py:184]: A relic from testing? Or if not, maybe use the logger?

**Review by PatrickMTonne — APPROVED** [review]
👍 Looks good!

**Review by haydn9000 — APPROVED** [review]
👍

File: bulk_site_creator/views.py (hunk 1 above)
Comment (Chris-Thornton-Harvard) [reply bulk_site_creator/views.py:184]: Yea, good catch

**Pull request merged**